packages/gafl-webapp-service/src/pages/renewals/renewal-inactive/__tests__/route.spec.js

```diff
@@ -1,45 +1,218 @@
 import { getData, getTitleAndBodyMessage } from '../route'
 import { NEW_TRANSACTION } from '../../../../uri.js'
 import { RENEWAL_ERROR_REASON } from '../../../../constants'
+import moment from 'moment-timezone'
 
-import englishTranslations from '../../../../locales/en.json'
-import welshTranslations from '../../../../locales/cy.json'
+const dateTimeDisplayMock = jest.requireMock('../../../../processors/date-and-time-display.js')
+jest.mock('../../../../processors/date-and-time-display.js', () => ({
+  cacheDateFormat: 'YYYY-MM-DD',
+  dateDisplayFormat: 'D MMMM YYYY'
+}))
 
-describe('renewal-inactive > route', () => {
-  const mockStatusCacheGet = jest.fn()
-
-  const mockRequest = {
-    cache: () => ({
-      helpers: {
-        status: {
-          getCurrentPermission: mockStatusCacheGet
-        }
+const englishTranslations = {
+  renewal_inactive_title_1: 'You are renewing this licence too early',
+  renewal_inactive_title_2: 'The licence renewal has expired',
+  renewal_inactive_title_3: 'You cannot renew an 8 day or 1 day licence',
+  renewal_inactive_not_due_1: 'The licence ending in ',
+  renewal_inactive_not_due_2: ' does not expire until ',
+  renewal_inactive_has_expired_1: ' has expired on ',
+  renewal_inactive_has_expired_2: ' and can no longer be renewed',
+  renewal_inactive_not_annual_1: ' is not a 12 month licence and cannot be renewed.'
+}
+
+const welshTranslations = {
+  renewal_inactive_title_1: 'You are renewing this licence too early',
+  renewal_inactive_title_2: 'The licence renewal has expired',
+  renewal_inactive_title_3: 'You cannot renew an 8 day or 1 day licence',
+  renewal_inactive_not_due_1: 'The licence ending in ',
+  renewal_inactive_not_due_2: ' does not expire until ',
+  renewal_inactive_has_expired_1: ' has expired on ',
+  renewal_inactive_has_expired_2: ' and can no longer be renewed',
+  renewal_inactive_not_annual_1: ' is not a 12 month licence and cannot be renewed.'
+}
```

Comment:
These are identical to `englishTranslations`, above, and as far as I can see nothing is asserted against either - so you could get rid of them and change `englishTranslations` to `sampleLabels`, or something like that. Better than that, however, might be moving the getMessages function below to here and using it whenever you need labels, not just for the block it's in.